lib/package.js

```diff
@@ -129,9 +79,9 @@ module.exports = function pkg(datastar, models) {
    *
    * @returns {Object} ?
    */
-  Pkg.fromPublish = function (payload) {
-    var replica = assign({}, this.extractLatest(payload));
-    var schemaFields = Pkg.schema.mappedFields();
+  function fromPublish({ pkg, payload }) {
+    var replica = assign({}, pkg.extractLatest(payload));
```

Comment:
extractLatest that is commented out is whats used here. And in this case, we actually just need the payload here, this is not a function that was on the prototype of the model, it was a static function attached to the class/constructor.
```suggestion
    var replica = assign({}, extractLatest(payload));
```